speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backend/DecisionMaker.inl

```diff
@@ -80,3 +88,40 @@ DecisionMaker<SocketBlackBox, SDAConfig>::~DecisionMaker()
     delete m_recorder;
 #endif
 }
+
+/// @brief When the race stops, the simulation data collected will be stored in the database
+template<typename SocketBlackBox, typename SDAConfig>
+void TEMP_DECISIONMAKER::RaceStop()
+{
+    std::string configPath(ROOT_FOLDER "\\data");
```

Comment:
I think it is better to move this function to SQLDataBaseStoreage, and let that class handle the functionality of logging in. Decisionmaker already has enough functionality as it is and logging into the database should not be added to that functionality

Reply:
Or make a new function in SQLDataBaseStorage

Reply:
I believe this is because eventually these parameters should be set by the front-end module and passed on from there. In which case perhaps add a comment explaining so, and perhaps still split it off to a separate function.

Reply:
Settings are currently set in a file, all the data is processed in the back-end, so the front-end has nothing to do with this